spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/cloudfoundry/reactive/ReactiveTokenValidatorTests.java

```diff
@@ -97,7 +98,7 @@ public void validateTokenWhenKidValidationFailsTwiceShouldThrowException()
 			throws Exception {
 		PublisherProbe<Map<String, String>> fetchTokenKeys = PublisherProbe
 				.of(Mono.just(VALID_KEYS));
-		ReflectionTestUtils.setField(this.tokenValidator, "cachedTokenKeys", VALID_KEYS);
+		ReflectionTestUtils.setField(this.tokenValidator, "cachedTokenKeys", new ConcurrentHashMap<>(VALID_KEYS));
```

Comment:
Couldn't `VALID_KEYS` be a `ConcurrentHashMap` instead?

Reply:
@wilkinsona I just changed there because `VALID_KEYS ` is also shared by `this.securityService.fetchTokenKeys()` in the tests. Is it better to make `VALID_KEYS` to `ConcurrentHashMap` for you?

Reply:
I think so, yes. It feels less noisy to not be wrapping `INVALID_KEYS` each time.